### DISCUSSION TITLE
Not getting any emails from Codemagic

### DISCUSSION THREAD

**Discussion by rtobescu** [codemagic-ci-cd/codemagic-docs#2243] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

I'm unable to receive any email form codemagic.io to a custom domain. This happens when I try to signup or when I use the email address in the recipients array of the publish field. I am able to receive emails to my gmail account.

### Expected results

To receive the emails

### Actual results

No email received

**Comment by dtrdic** [comment, marked as answer]
Hey @rtobescu I suspect the email is not arriving due to your company anti spam/security etc. Can you check your spam/junk folder maybe? But you can log in via OAuth right?

**Reply by rtobescu** [reply]
Thank you for the help, you are correct, it was our mail provider that completely dropped your email without even sending them into spam. After adding codemagic.io to the whitelist we've started receiving them. I have not tried connecting with any of identity providers.
